src/routes.tsx

```diff
@@ -104,7 +105,7 @@ const pathRoutes: Path[] = [
   },
   {
     path: '/namespaces/:namespace/' + Paths.APPLICATIONS + '/:app',
-    component: lazy(() => import('./pages/AppDetails/AppDetailsPage'))
+    component: AppDetailsPage
```

Comment:
I don't understand why the details page wouldn't lazy load. That is the page that one would think you would want to lazy load. Must be some shared component somewhere.

Reply:
Yes, most likely. That will be somewhat hard to find.